CMakeLists.txt

```diff
@@ -218,7 +218,7 @@ endif()
 
 # if SEND_ODOMETRY_DATA option is set, then full odometry data is sent instead of
 # only the visual pose estimate
-set(send_odometry "false")
+set(send_odometry "true")
```

Comment:
This needs to be set through `SEND_ODOMETRY_DATA` option set in build time on the Firmware side, similar to `SEND_VISION_ESTIMATION_DATA`. If one is set, the other should be unset though.

Reply:
Got it.